plutus-tx-plugin/src/PlutusTx/Plugin.hs

```diff
@@ -335,7 +335,7 @@ runCompiler opts expr = do
     when (poDoTypecheck opts) . void $
         liftExcept $ PLC.typecheckPipeline plcTcConfig plcP
 
-    uplcP <- liftExcept $ UPLC.deBruijnProgram $ UPLC.eraseProgram plcP
+    uplcP <- liftExcept $ UPLC.deBruijnProgram $ UPLC.simplifyProgram $ UPLC.eraseProgram plcP
```

Comment:
If we had multiple simplifications/optimisations, would there be any point in adding something to let you turn individual things on and off?  I suppose that might be useful for testing the effects of different things, but it's probably not something you'd want to use in production.

Reply:
I think we can add this later if we need it!